tailcfg/tailcfg.go

```diff
@@ -1897,39 +1892,88 @@ type TKAInitFinishRequest struct {
 // key authority.
 type TKAInitFinishResponse struct{}
 
-// TKAMapRequest describes request parameters relating to the tailnet key
-// authority instance on this node. This information is transmitted as
-// part of the MapRequest.
-type TKAMapRequest struct {
-	// Head is the AUMHash of the latest authority update message committed
-	// by this node.
-	Head string // tka.AUMHash.String
-}
-
-// TKAMapResponse encodes information for the tailnet key authority
-// instance on the node. This information is transmitted as
-// part of the MapResponse.
-//
-// If there are no updates to be transmitted (in other words, if both
-// control and the node have the same head hash), len(Updates) == 0 and
-// WantSync is false.
-//
-// If control has updates that build off the head hash reported by the
-// node, they are simply transmitted in Updates (avoiding the more
-// expensive synchronization process).
-//
-// In all other cases, WantSync is set to true, and the node is expected
-// to reach out to control separately to synchronize.
-type TKAMapResponse struct {
-	// Updates is any AUMs that control believes the node should apply.
-	Updates []tkatype.MarshaledAUM `json:",omitempty"`
-
-	// WantSync is set by control to request the node complete AUM
-	// synchronization.
+// TKAInfo encodes controls' view of tailnet key authority (TKA) state.
```

Comment:
It would be "control's" (it's not a possessive of a plural / ending in -s, etc)

But normally we don't say just "control" in oss docs and prefer "the control plane" or "the coordination server". It's not entirely consistent.

Reply:
Im bad at english :joy: :joy: I didnt know the rule for possession with trailing apostrophe required it to end in -s. I'll fix lol